sale_order_lot_selection/models/sale.py

```diff
@@ -47,9 +47,10 @@ def get_move_from_line(self, line):
                 if move_line_id and line.lot_id == move_line_id[:1].lot_id:
                     move = m
                     lot_count += 1
-                    # if counter is 0 or > 1 means that something goes wrong
-                    if lot_count != 1:
-                        raise UserError(_('Can\'t retrieve lot on stock'))
+                    # if counter is 0 means that something goes wrong
+                    if lot_count == 0:
+                        raise UserError(_('Can\'t retrieve product stock %s, for lot %s') %
```

Comment:
I read again your code.
It seems that `loc_count` can't be equals to 0 because in line 49 it's set from 0 to 1 or upper.
Exception is never raised.

But I need to solve this case. I'll try to fix in another PR. Thanks